lib/spack/spack/spec.py

```diff
@@ -81,18 +81,16 @@
 import collections
 import hashlib
 import itertools
+import operator
 import os
 import re
 
 import six
-
-from operator import attrgetter
```

Comment:
I fully agree with these changes. In general I would always import a "namespace", be it the default one or an alias like `fs` or `lang` just below. The only exception to this rule I generally make is if I use relative imports for modules in a package that are not supposed to be accessed directly by clients, but that happens rarely if at all in Spack.